vuepress/docs/ecr/tutorials/ecr-deploy-use-plugin-and-mfe-without-bundle.md

```diff
@@ -0,0 +1,170 @@
+<span id="section2"></span>
+
+To populate the bundle with the generated micro frontends, run the
+`./buildBundle.sh` script or use the npm.
+
+    npm run populate-bundle
+
+> **Important**
+>
+> The bundle population with the micro frontends requires some time to
+> be processed. You should be able to follow the progress of the
+> operation on screen.
+
+## 3. Generate a docker image for your microservice
+
+JHipster uses the JIB Maven plugin to generate a docker image for your
+microservice.
+
+By default the output image will use the name of your application and
+the `0.0.1-SNAPSHOT` tag, eg. `entando/jhipster@0.0.1-SNAPSHOT`.
+
+    ./mvnw -Pprod clean package jib:dockerBuild [-Djib.to.image=<your-image-name]
+
+> **Note**
+>
+> By default, the organization used for the image is `entando` but you
+> can simply change it in the `pom.xml` file or by providing the
+> `-Djib.to.image=<org>/<name>:<version>` to the jib:dockerBuild
+> command.
+
+> **Note**
+>
+> Output image name is controlled in the `pom.xml` file but could also
+> be customized in the `./mvnw` command using the `-Djib.to.image`
+> parameter.
+
+> **Warning**
+>
+> If you change the target image of the docker build, remember to update
+> the plugin metadata in the bundle accordingly.
+
+## 4. Publish the Docker image to Docker registry (DockerHub or equivalent)
+
+Let’s now publish the docker image for the microservice to make it
+available later during bundle installation in the cluster.
+
+    docker push <name-of-the-image:tag>
+ 
+ ## 5. Deploy the plugin in your Entando Cluster
+ 
+ Assuming you have an Entando Cluster available in an `entando` namespace, you can now take
+ the plugin custom resource generated by the Entando Blueprint in the `bundle/plugins` folder and deploy it 
+ in your `entando` namespace. 
+ 
+> **Warning**
+>
+> As states at step 4, if you changed the target image of your docker build, the plugin custom resource
+> in the `bundle/plugins` folder needs to be updated to point to the correct image 
+
+From the jhipster project root
+
+```$bash 
+cd bundle/plugins
+
+kubectl create -f <plugin-file.yaml> -n entando
+```
+ 
+ ## 6. Wait for you plugin to be in `Running` state and link the plugin with the app using an `EntandoAppPluginLink` custom resource
+
+Once the plugin server deployment will be up and running, you can create an EntandoAppPluginLink custom resource
+to make the plugin API available from the EntandoApp domain. 
+Here an example of a EntandoAppPluginLink custom resource. Some assumptions with this custom resource:
+- The EntandoPlugin generated with the blueprint has name `my-demo-plugin`
+- The EntandoApp where I want to expose the `my-demo-plugin` apis is named `my-entando-app` 
+- Both are deployed on the `entando` namespace. 
+- As a name for the link, I'll use `my-entando-app-to-my-demo-plugin-link`
+
+> **Warning**
+> Remember to change the fields to match your setup.
+
+```
+> cat <<EOF > /tmp/mylink.yaml
```

Comment:
Do we describe this CR structure elsewhere in the docs?

Reply:
Yeah, in the custom-resources document. I'll add a link to that document